experiments/AMIP/modular/components/ocean/slab_ocean_init.jl

```diff
@@ -32,6 +32,7 @@ struct OceanSlabParameters{FT <: AbstractFloat}
     z0m::FT
     z0b::FT
     α::FT
+    evolving_switch::FT
```

Comment:
Is this different from the `evolving_ocean` Boolean arg ?

Reply:
Seems so : I see that we have an arg `evolving` which reads the `CLI` option; and then assigns it to the `evolving_switch`

Reply:
It's related. `evolving_switch` stores 1 or 0, so we don't have to do the `weight = evolving ? : 1 : 0` statement in the rhs function.